### DISCUSSION TITLE
Add ability to use smoke with disabled mouse buttons

### DISCUSSION THREAD

**Discussion by Uniter343** [ppy/osu#20623] [Ideas] [answered]
Currently when you have mouse buttons disabled you can't use them at all. If smoke is bound to one of them you can not use it. It would be better to only disable actual inputs / still allow smoke with the setting turned on.

**Comment by peppy** [comment, marked as answer]
If you basically never want to use mouse buttons, you can just unbind them instead (instead of using the "disable mouse buttons" toggle).
